packages/medusa/src/services/cart.ts

```diff
@@ -1804,6 +1811,11 @@ class CartService extends TransactionBaseService {
                   return this.paymentProviderService_
                     .withTransaction(transactionManager)
                     .createSession(paymentSessionInput)
+                    .catch((e) => {
+                      this.logger_.warn(
+                        `Unable to create the payment session for ${paymentProvider.id}${EOL}${e.stack}`
```

Comment:
comment: Ideally, we would also gracefully handle sessions that fail to update. I realise the case it a bit more tricky, as it could lead to stale session data. Have you thought of a way of handling this? One way would be to delete the session entirely.

Reply:
I did not want to touch the update for this exact reason. We can discuss it